dune_daq_services/ers/ers-protobuf-dbwriter/kustomization.yml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: {{ DUNE_ers.namespace }}
+
+resources:
+  - ersdbwriter.yaml
+
+patches:
+  - target:
+      group: apps
+      version: v1
+      kind: Deployment
+    patch: |-
+      - op: replace
+        path: /spec/template/spec/containers/0/image
+        value: {{ dunedaq.unified_microservices.image }}:{{ dunedaq.unified_microservices.tag }}
```

Comment:
This is where the image tag/version is selected.  You may want to tweak it here.